DEVELOPMENT.md

```diff
@@ -0,0 +1,67 @@
+## Development
+
+### Code Style
```

Comment:
We should mention somewhere: no abbreviations, slang or inside jokes as this make is harder for non-native english speaker to understand the code.